.github/workflows/test_integration_jaas.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
   # Ensure project builds before running test
   build:
     name: Build-JAAS
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, jammy, x64]
```

Comment:
The build jobs should all run on GitHub runners, not self-hosted. They are too short and will delay the over all time.